src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java

```diff
@@ -0,0 +1,192 @@
+
+  private SignInPage openLoginModalOnDesktop() {
+    return new DetachedSignInPage(new NavigationBar().clickOnSignIn());
+  }
+
+  @Test
+  public void anonymizedUserCannotLogin() {
+
+    Credentials credentials = new Credentials();
+    String timestamp = Long.toString(DateTime.now().getMillis());
+    String qanon = "QAanon" + timestamp;
+    String passw = "makak";
+    SignUpUser
+        user = new SignUpUser(qanon, credentials.emailAnonymousUserTestWikia, passw,
+                              LocalDate.of(1990, 3, 19));
+    UserRegistration.registerUserEmailConfirmed(user);
+
+    SpecialAnonymizationUserPage anonymizationStaff = new SpecialAnonymizationUserPage().open();
+    anonymizationStaff.loginAs(User.STAFF);
+    anonymizationStaff.fillFutureAnon(qanon)
+        .submitAnonymization();
+
+    Assertion.assertStringContains(anonymizationStaff.getAnonConfirmation(), qanon);
+
+    anonymizationStaff.logOut();
+
+    SignInPage signIn = openLoginModalOnDesktop();
+    signIn.login(qanon, passw);
+    Assertion.assertEquals(signIn.getError(), ERROR_MESSAGE);
+  }
+
+  @Test
+  public void anonymizedUserProfilePageRemoved() {
+
+    Credentials credentials = new Credentials();
+    String timestamp = Long.toString(DateTime.now().getMillis());
+    String qanon = "QAanon" + timestamp;
+    String passw = "makak";
+    SignUpUser
+        user = new SignUpUser(qanon, credentials.emailAnonymousUserTestWikia, passw,
+                              LocalDate.of(1990, 3, 19));
+    UserRegistration.registerUserEmailConfirmed(user);
+
+    SpecialAnonymizationUserPage anonymizationStaff = new SpecialAnonymizationUserPage().open();
+    anonymizationStaff.loginAs(User.STAFF);
+    anonymizationStaff.fillFutureAnon(qanon)
+        .submitAnonymization();
+
+    Assertion.assertStringContains(anonymizationStaff.getAnonConfirmation(), qanon);
+
+    UserProfilePage userProfilePage = new UserProfilePage().open(qanon);
+    Assertion.assertTrue(userProfilePage.getUserNotExists().equals(String.format("User account \"%s\" does not exist or has never logged in on this wiki.", qanon)));
+  }
+
+  @Test
+  public void anonymizationWikiEdits() {
+
+    Credentials credentials = new Credentials();
+    String timestamp = Long.toString(DateTime.now().getMillis());
+    String qanon = "QAanon" + timestamp;
+    String passw = "makak";
+    SignUpUser
+        user = new SignUpUser(qanon, credentials.emailAnonymousUserTestWikia, passw,
+                              LocalDate.of(1990, 3, 19));
+    UserRegistration.registerUserEmailConfirmed(user);
+
+    new WikiBasePageObject().loginAs(user.getUsername(), user.getPassword(), wikiURL);
+
+    new ArticleContent(user.getUsername()).push("Test" + timestamp, "AnonymizationTest");
+
+    ArticleHistoryPage articleHistoryPage = new ArticleHistoryPage().open("AnonymizationTest");
+    String id = articleHistoryPage.getHistoryID(user.getUsername());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "id" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)